core/baseline-core-v1.0-psd01.md

```diff
@@ -2946,14 +3013,20 @@ This document lists the requirements for either option and indicates the differe
 * *Support for database clustering to process high data volumes in short periods of time (sub second processing)*
 * *Security features that adhere to established industry security standards such as the US Federal Information Processing Standard [[FIPS](#fips)] or [[ISO 27001](#iso27001)]*  
 
+[[R309]](#r309) Testability: An example of a BPI consistent with the requirement is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite in line with enterprise-grade data base solutions, the requirement is testable.
+
 #### **[O8]** 
 Centralized BPI Storage MAY be partially persistent. 
 
+[[O8]](#o8) Testability: An example of centralized BPI Stroage that can be made partially consistent by restricing writes to a row in a table to one time is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite, the requirement is testable.
+
 #### **[CR34]>[O8]** 
 Partially Persistent BPI Storage  MUST be append-only.
 
 *Non-normative examples of such data bases are, but not limited to, OracleDB, MongoDB, PostGres, Cassandra, and DynamoDB.*
 
+[[CR34]>[O8]](#cr34o8) Testability: An example of centralized BPI Stroage that is append-only by restricing writes to a row in a table to one time is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite, the requirement is testable.
```

Comment:
```suggestion
[[CR34]>[O8]](#cr34o8) Testability: An example of centralized BPI Storage that is append-only by restricting writes to a row in a table to one time is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite, the requirement is testable.
```